GOVERNANCE.md

```diff
@@ -43,23 +54,24 @@ depend on and trust to make decisions in the best interest of the project.
 
 Periodically, the existing maintainers curate a list of contributors that have
 shown regular activity on the project over the prior months. From this list,
-maintainer candidates are selected and proposed on the maintainers mailing list.
+maintainer candidates are selected and proposed in the maintainers forum.
 
-After a candidate has been announced on the maintainers mailing list, the
+After a candidate has been proposed in the maintainers forum, the
```

Comment:
s/been/been informally/

Reply:
i think it's fair to have a discussion first in the forum.. then after some obvious threshold go ahead and open the PR starting the voting window.